utils/eui.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 Stichting ThingsIX Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"encoding/binary"
+	"github.com/brocaar/lorawan"
+)
+
+// Eui64ToUint64 converts a lorawan.EUI64 into an uint64 in BigEndian format.
+func Eui64ToUint64(eui64 lorawan.EUI64) uint64 {
+	return binary.BigEndian.Uint64(eui64[:])
+}
+
+// Eui64FromString tries to read a lorawan.EUI64 from string. It returns an error if it doesn't succeed.
+func Eui64FromString(str string) (*lorawan.EUI64, error) {
```

Comment:
We can move the functions in `thingsix.go` to this file since they also convert from/to EUI64, only on byte slices.